input/data/2014/2014MNRAS.439.2828A/tev-000088.yaml

```diff
@@ -1,4 +1,23 @@
 source_id: 88
 reference_id: 2014MNRAS.439.2828A
 
-# TODO: add data
+data:
+  livetime: 63.4
+
+pos:
+  ra: {val: 16h40m41.0s, err: 1.0s, err_sys: 1.3s}
+  dec: {val: -46d32m31s, err: 14s, err_sys: 20s}
+
+morph:
+  type: gauss
+  sigma: {val: 4.3m, err: 0.2m}
+  sigma2: {val: 4.3m, err: 0.2m}
```

Comment:
Remove `sigma2` that duplicates the value of `sigma`.
The idea is that we only add `sigma2` for elongated sources, and here a symmetric Gaussian was fitted.
We could make our schema more explicit in the future, putting `gauss_elongated` or something, but for now this is how other sources were done.